docs/modules/ROOT/pages/customizing-entities/entity-views.adoc

```diff
@@ -10,9 +10,9 @@ The following section gives an overview of common customizations for generated e
 
 == Changing entity names, property names or other labels
 All entity names, property names and labels can be customized using message sources.
-For an explanation of the different message codes used, see <<appendix-message-codes,the relevant appendix>>.
```

Comment:
I would not remove the reference to the message codes. 
Labels can either be static text labels, such as the label of a formGroup, but they can also be labels for an instance of an object type. For example a user would be rendered with its display name as label.